src/config.ts

```diff
@@ -57,6 +57,7 @@ export const ENABLE_QUERY_BATCHING: any = false
 export const ENABLE_SAVED_SEARCH: any = false
 export const ENABLE_SERVER_SIDE_CACHE: any = false
 export const ENABLE_WEB_CRAWLING: any = false
+export const ENABLE_YOUR_ALERTS_PAGE: any = false
```

Comment:
**question:** Is there any reason that we use explicit `any` as type everywhere in this file 🤔 it is a more of a force specific question

Reply:
Good question. But unfortunately I won't be able to answer it 🤷‍♂️

Reply:
No worries, feel free to merge from my side